server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -226,8 +234,14 @@ public AsyncJobStatus queueJob(JobBuilder builder) {
             status = this.postJobStatusMessage(status);
         }
         catch (JobStateManagementException e) {
-            log.error("Unable to update state for job: {}; leaving job in its previous state for " +
-                "state resync upon execution", status.getName());
+            if (log.isDebugEnabled()) {
+                log.error("Unable to update state for job: {}; leaving job in its previous state for " +
+                    "state resync upon execution", status.getName(), e);
+            }
+            else {
+                log.error("Unable to update state for job: {}; leaving job in its previous state for " +
```

Comment:
It's for noise reduction, as the stack trace probably isn't necessary in the general case. This looks weird since both are error-level, but one doesn't output the stack trace.